source/System.cpp

```diff
@@ -353,6 +353,24 @@ void System::Load(const DataNode &node, Set<Planet> &planets)
 					grand.PrintTrace("Warning: Skipping unsupported arrival distance limitation:");
 			}
 		}
+		else if (key == "departure" && hasValue)
```

Comment:
Does this work for the "specific distances" syntax, because there I expect hasValue to be false if the values are given specifically for link and for jump.

Reply:
fixed